tests/modules/rtgtools/vcfeval/test.yml

```diff
@@ -0,0 +1,21 @@
+- name: rtgtools vcfeval test_rtgtools_vcfeval
+  command: nextflow run tests/modules/rtgtools/vcfeval -entry test_rtgtools_vcfeval -c tests/config/nextflow.config
+  tags:
+    - rtgtools
+    - rtgtools/vcfeval
+  files:
+    - path: output/rtgtools/test_results.txt
+      md5sum: 2e011aa6e54d258fcc3b45b2dda02ae4
+    - path: output/rtgtools/versions.yml
+      md5sum: 270ed7a5a8e347b251eb4aa2198f98e8
+
+- name: rtgtools vcfeval test_rtgtools_vcfeval_no_index
+  command: nextflow run tests/modules/rtgtools/vcfeval -entry test_rtgtools_vcfeval_no_index -c tests/config/nextflow.config
+  tags:
+    - rtgtools
+    - rtgtools/vcfeval
+  files:
+    - path: output/rtgtools/test_results.txt
+      md5sum: 2e011aa6e54d258fcc3b45b2dda02ae4
+    - path: output/rtgtools/versions.yml
+      md5sum: 8d0407000988c78fa43fe5cfe3d4449d
```

Comment:
versions.yml shouldn't have a md5sum. (It is already removed in the newest dev version of tools)
```suggestion
```